server/views/partials/feedbackBanner.njk

```diff
@@ -1,6 +1,7 @@
 {% if feedbackSurveyUrl %}
   <aside class="app-feedback-banner govuk-!-display-none-print">
     Non-associations is a new service.
-    Help improve it by <a href="{{ feedbackSurveyUrl }}" target="_blank">telling us what you think (opens in a new tab)</a>.
+    Share your feedback in this <a href="{{ feedbackSurveyUrl }}" target="_blank">2 question survey (opens in a new tab).</a>
```

Comment:
i would do `</a>.` instead of `.</a>` so that the dot isn't part of the link text

Reply:
oh and integration tests need a tweak somewhere